src/main/controlMod.F90

```diff
@@ -294,7 +292,11 @@ subroutine control_init( )
     ! Set clumps per procoessor
 
 #if (defined _OPENMP)
-    clump_pproc = omp_get_max_threads()
+!$omp parallel
+!$omp critical
```

Comment:
One minor change: I'll probably change this to `omp master` rather than `omp critical`. It shouldn't matter in practice, but that makes it more clear to the reader that this only needs to be done by a single thread.